packages/strapi-plugin-users-permissions/config/functions/bootstrap.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+/**
+ * An asynchronous bootstrap function that runs before
+ * your application gets started.
+ *
+ * This gives you an opportunity to set up your data model,
+ * run jobs, or perform some special logic.
+ */
+
+module.exports = cb => {
+  strapi.plugins['users-permissions'].services.userspermissions.updatePermissions();
```

Comment:
Is the callback returned after it means the `updatePermissions()` is synchronous? I suppose we could make asynchronous especially with `fs`. Don't forget that these functions are running before the server starts. If every plugin has a `bootstrap` function we are going to increase the boot time seriously... After all the efforts we put to reduce it to his minimum, I'm not a big fan of this idea.

I would definitely prefer something like this :)
```js
strapi.plugins['users-permissions'].services.userspermissions.updatePermissions(cb);
```